docs/ui/components/Home/sections/QuickStart.tsx

```diff
@@ -95,30 +144,30 @@ export function QuickStart() {
         </GridCell>
         <GridCell
           className={mergeClasses(
-            'relative z-0 min-h-[172px] border-palette-purple6 bg-palette-purple3',
-            'selection:bg-palette-purple5',
+            'relative z-0 min-h-[172px] border-palette-orange6 bg-palette-orange3',
```

Comment:
@simek, I went with a purple shade from our style guide for test flight deployment, and since there were two purple tiles, I decided to change this one to use a different palette. Let me know if that doesn't work, I'll revert this.

Reply:
Purple looks nice, seems good to me! :)